labman/db/tests/tester.py

```diff
@@ -416,8 +416,8 @@ def stress_tests(num_plates):
 ReverseComplement,0
 
 \[Data\]
-Sample_ID,Sample_Name,Sample_Plate,Sample_Well,I7_Index_ID,index,Sample_Project,Description,,
-New_sequencing_run_\d{4}-\d{2}-\d{2}_\d{2}_\d{2}_\d{2}_\d{6},,,,,NNNNNNNNNNNN,,,,,"""  # noqa
+Sample_ID,Sample_Name,Sample_Plate,Sample_Well,I7_Index_ID,index,Sample_Project,Well_Description,,
+New_pool_name_\d{4}-\d{2}-\d{2}_\d{2}_\d{2}_\d{2}_\d{6},,,,,NNNNNNNNNNNN,,\d+,,,"""  # noqa
```

Comment:
What's the `\d+` for?

Reply:
`\d+` is the `composition_id` property of a pool, which is a number (e.g., `3079`). During yesterday's discussion we decided to let "Well_Description" be the identifier of the pool.